### DISCUSSION TITLE
Backups directory size is out of control

### DISCUSSION THREAD

**Discussion by AdamLMark** [lloesche/valheim-server-docker#643] [Q&A]
I just discovered my Docker host had 700 GB of Valheim backups!

My `world.db` is 50 MB. There used to be 4 files in the `worlds_local` directory (100 MB), but now there were 100+ `.db` and `.fwl` files like these:
```
world_backup_auto-20230921153149.db
world_backup_auto-20230921153149.fwl
world_backup_20230822-084820.db
world_backup_20230822-084820.fwl
```

This made the zipped hourly backups 2.5 GB each!

The only non-default backup variable I set was for 30-day retention.

I didn't see any other issues or discussions on this, but I can't be the only one experiencing it!

Cheers 🍻

**Comment by Concretv** [comment]
Hi ! Same on my side,
I have 2 new backup files generated each day (db and fwl) in the "config>world_local" directory then 1 ziped file generated every hour in the "config>backup" (containing the "world_local" files which increase every day and leads to larger ziped backup file)
How can we remove the backup files in the "world_local" directory to keep only the zipped backup files in the "backup" directory (containing only one db and one fwl file) ?
